.coderabbit.yaml

```
@@ -0,0 +1,79 @@
+  collapse_walkthrough: false
+  changed_files_summary: true
+  sequence_diagrams: true
+  assess_linked_issues: true
+  related_issues: true
+  related_prs: true
+  suggested_labels: true
+  auto_apply_labels: true  # Auto-apply suggested labels
+  suggested_reviewers: true
+  auto_assign_reviewers: true  # Auto-assign reviewers
+  poem: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)